asserts/assertstest/assertstest.go

```diff
@@ -159,29 +163,28 @@ func NewAccount(db SignerDB, username string, otherHeaders map[string]interface{
 		otherHeaders["account-id"] = strutil.MakeRandomString(32)
 	}
 	if otherHeaders["display-name"] == nil {
-		otherHeaders["display-name"] = strings.ToTitle(username)
+		otherHeaders["display-name"] = strings.ToTitle(username[:1]) + username[1:]
```

Comment:
Huh.. I didn't notice that logic here before. Can we please drop display-name altogether while we don't have proper content for it?  Automating it like this is no good.